pkg/core/resources/store/store.go

```diff
@@ -143,12 +149,35 @@ func IsResourceNotFound(err error) bool {
 	return err != nil && strings.HasPrefix(err.Error(), "Resource not found")
 }
```

Comment:
I would remove this function after introducing separate  `ResourceNotFoundError`